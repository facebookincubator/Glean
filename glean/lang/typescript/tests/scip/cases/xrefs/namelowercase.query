query: lsif.NameLowerCase _
transform: [gensort, []]
