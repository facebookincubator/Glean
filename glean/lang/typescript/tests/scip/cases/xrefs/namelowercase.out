[
  "@generated",
  { "key": { "nameLowerCase": "", "name": { "key": "" } } },
  { "key": { "nameLowerCase": "error", "name": { "key": "Error" } } }
]